pkg/controller/config/config_controller.go

```diff
@@ -233,6 +256,17 @@ func (r *ReconcileConfig) reconcileInstall(req reconcile.Request, res *op.Config
 			Version: tektonVersion})
 		return reconcile.Result{}, err
 	}
+
+	if err := r.installAddons(req, res, tfs); err != nil {
```

Comment:
wouldn't it be better if we move the installAddon logic into a reconileAddons function